api/PclusterApiHandler.py

```diff
@@ -685,9 +690,59 @@ def _get_params(_request):
     params.pop("path")
     return params
 
+def check_tags():
+    try:
+        cost_explorer = boto3.client('ce',)
+        inactive_tags = cost_explorer.list_cost_allocation_tags(Status='Inactive',TagKeys=PCLUSTER_COST_TAGS)
+        return {'TagStatus': not (len(inactive_tags['CostAllocationTags']) > 0)}
+    except Exception as e:
+        return {"message": str(e)}, 500
 
-# Proxy
+def activate_tags():
+    try:
+        cost_explorer = boto3.client('ce')
+        newlist = [{'TagKey': tag, 'Status': 'Active'} for tag in PCLUSTER_COST_TAGS]
+        cost_explorer.update_cost_allocation_tags_status(CostAllocationTagsStatus=newlist)
+        return "Activated Tags"
+    except Exception as e:
+        return {"message": str(e)}, 500
 
+def get_graph_data():
+    try:
+        cluster_name = request.args.get("cluster_name")
+        Start = request.args.get("Start")
+        End = request.args.get("End")
```

Comment:
Suggestion:
the only thing I would add here is a check to make sure all these three values are actually provided
if any of these is `None`, than I would return an error 400 bad request since the request would be improper